src/components/edit/EditMishna/FormCommentExcerpt.tsx

```diff
@@ -72,7 +72,7 @@ const FormCommentExcerpt: FC<IProps> = ({
         blocks: [{ ...excerpt.editorStateFullQuote.blocks[0], text: values.comment }],
       },
     };
-    saveExcerpt(getTractate(mishna), getChapter(mishna), mishna.mishna, excerptToSave);
```

Comment:
if you are not using `getTractate` and `getChapter` you can remove them completely from mishnaUtils

Let's just make sure that the behavior is the same all across the code and these values are not taken differently